lld/test/wasm/page-size.s

```diff
@@ -0,0 +1,48 @@
+# RUN: llvm-mc -filetype=obj -triple=wasm32-unknown-unknown %s -o %t.o
+# RUN: wasm-ld -no-gc-sections -o %t.wasm %t.o --page-size=1
+# RUN: obj2yaml %t.wasm | FileCheck %s
+
+    .section .data.foo,"",@
+    .globl  foo
+    .hidden  foo
+    .p2align        2
+foo:
+    .int32  0xffffffff
+    .size   foo, 4
+
+    .section .text._start,"",@
+    .globl  _start
+_start:
+    .functype _start () -> (i32)
+    i32.const __wasm_first_page_end
```

Comment:
You can do `i32.const foo` here instead of `-no-gc-sections`... not sure which is better?

Reply:
Soft preference towards no-gc-sections but if you feel differently I am happy to change it.